### ISSUE TITLE
Marketing: Create a recognizable logo

### ISSUE THREAD

**Issue by TomCools** [belgian-jug/belgian-jug.github.io#3]
The current logo on Twitter and Meetup isn't very "Java" recognizable. 
We should change this.

**Comment by lmagnette** [comment]
@TomCools I enjoy drawing in my free time, so after seeing this, I gave it a try.  Tell me if it's near anything that can match what you're looking for.  Let me know your thoughts and feedback.
![beJug](https://user-images.githubusercontent.com/6390187/236571629-55ada663-1eb9-4010-820e-bc7b97a0ca61.png)

**Comment by TomCools** [comment]
@lmagnette I love it! Mind if we throw it on the Twitter/Mastodon to see what the rest of the community thinks? 😊

**Comment by lmagnette** [comment]
@TomCools I'm happy to hear that you love it. No problem to me if you to share it and get some more feedback

**Comment by FDelporte** [comment]
Love it too! One small remark, maybe a small line or shadow is needed for the Duke's arms to make them stand out from the black background?

**Comment by TomCools** [comment]
@lmagnette, I heard a lot of good noises! People like it! Any idea when you'll have the time to look into @FDelporte's comment? :-)

**Comment by lmagnette** [comment]
@TomCools here's an updated version based on @FDelporte remark.

![bejug-2023-05-14](https://github.com/BeJUG/bejug.github.io/assets/6390187/5654b0f1-d808-4635-a1c5-b40372b46f2b)

**Comment by TomCools** [comment]
That was quick! :) @FDelporte what do you think?

**Comment by SimonVerhoeven** [comment]
It's certainly clearer, mayhaps also give the french fries the white outline?

**Comment by lmagnette** [comment]
@SimonVerhoeven you're right I forgot the white outline on the french fries. Here's the updated version.
![bejug-2023-05-14](https://github.com/BeJUG/bejug.github.io/assets/6390187/eb9b7d5b-d212-48a9-859a-e92177d10376)

**Comment by FDelporte** [comment]
Love it :-)

**Comment by SimonVerhoeven** [comment]
I like it.

**Comment by TomCools** [comment]
🚢  to production then! 😊 🚀
